### ISSUE TITLE
How do you determine the central latitude/longitude values used in the examples?

### ISSUE THREAD

**Issue by BlackArbsCEO** [ResidentMario/geoplot#52]
Example from [here](https://residentmario.github.io/geoplot/examples/nyc-collision-factors.html): 

```

# Load the data (uses the `quilt` package).
import geopandas as gpd
from quilt.data.ResidentMario import geoplot_data

boroughs = gpd.read_file(geoplot_data.nyc_boroughs())
collisions = gpd.read_file(geoplot_data.nyc_collision_factors())


# Plot the data.
import geoplot.crs as gcrs
import geoplot as gplt
import matplotlib.pyplot as plt

fig = plt.figure(figsize=(10,5))

ax1 = plt.subplot(121, projection=gcrs.AlbersEqualArea(central_latitude=40.7128, central_longitude=-74.0059))
```
Where do those `central_latitude` and `central_longitude` values come from?

**Comment by ResidentMario** [comment]
I explain why I'm doing this in the [Projections section of the tutorial](https://nbviewer.jupyter.org/github/ResidentMario/geoplot/blob/master/notebooks/tutorials/Projections.ipynb). Basically every projection has a center point that you have to specify. If you don't specify one explicitly, `geoplot` will take an educated guess using the centroid of the data. That'll be ~right, but it won't "look" quite right because it will be slightly off.

Bottom line is that for prettiest-possible looking plot, you should tweak the central coordinates yourself. This example does that. As for where I found it...pretty sure I dropped a pin in Google Maps!
